src/main/resources/db/migration/V1_28__update_skill_table.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO SKILL(skill_id,skill_code,description,service_id,user_type,created_date,last_update) VALUES
+	 (42,'SKILL:AAA7:DAMAGES_CCMC_CMC_MANAGEMENT','CCMC/CMC Management','AAA7','Future Operations',timezone('utc', now()),timezone('utc', now()));
+INSERT INTO SKILL(skill_id,skill_code,description,service_id,user_type,created_date,last_update) VALUES
+	 (43,'SKILL:AAA6:MONEY_CLAIMS_CCMC_CMC_MANAGEMENT','CCMC/CMC Management','AAA6','Future Operations',timezone('utc', now()),timezone('utc', now()));
+INSERT INTO SKILL(skill_id,skill_code,description,service_id,user_type,created_date,last_update) VALUES
+	 (44,'SKILL:AAA7:DAMAGES_MULTI_TRACK_TRIAL_MANAGEMENT','Multi Track Trial Management','AAA7','Future Operations',timezone('utc', now()),timezone('utc', now()));
```

Comment:
should it be `Multi and Intermediate Track Trial Management` instead of `Multi Track Trial Management` for the last 2 scripts?

Reply:
I have updated the skill descriptions to match the Jira and double checked all of the other fields/data.